### PR TITLE
Jdr garnett opinion

### PR DESCRIPTION
**Pull request by Joshua-David**
## What does this change?
GT opinion

## What is the value of this and can you measure success?
N/A

## Does this affect other platforms - Amp, Apps, etc?
N/A

## Does this affect GLabs Paid Content Pages? Should it have support for Paid Content?
N/A

## Screenshots
N/A

## Tested in CODE?
Yez

### CODE REVIEW

File: static/src/stylesheets/module/content-garnett/_types.scss [hunk 1]
```diff
@@ -272,9 +312,17 @@ $quote-tail: 25px;
...
+        @include mq(phablet) {
+            padding-bottom: 0;
+        }
         @include mq(tablet) {
             @include fs-headlineGarnett(6);
             font-weight: 500;
+            padding-bottom: 1.5rem;
+        }
+        @include mq(leftCol) {
+            // padding-bottom: 2.5rem;
```
Comment (sndrs) [inline static/src/stylesheets/module/content-garnett/_types.scss]: unneeded?

File: static/src/stylesheets/module/content-garnett/_types.scss [hunk 2]
```diff
@@ -499,3 +553,112 @@ $quote-tail: 25px;
...
+        }
+        @include mq(phablet) {
+            padding-left: 0;
+        }
+        @include mq(leftCol) {
+            padding-top: $gs-baseline;
+        }
+    }
+
+    .content__headline--byline span a {
```
Comment (sndrs) [inline static/src/stylesheets/module/content-garnett/_types.scss]: is the `span` needed here?

**Review by sndrs — APPROVED** [review]
looks good other than the comments!

File: static/src/stylesheets/module/content-garnett/_types.scss (hunk 2 above)
Comment (Joshua-David) [reply static/src/stylesheets/module/content-garnett/_types.scss]: yea is targeted that way, doesn't override otherwise

**Pull request merged**